src/platform/user/authentication/utilities.js

```diff
@@ -39,11 +39,39 @@ export function clearRavenLoginType() {
   Raven.setTagsContext({ loginType: undefined });
 }
 
+function redirectWithGAClientId(redirectUrl) {
+  try {
+    let clientId;
+
+    // eslint-disable-next-line
+    const trackers = ga.getAll();
+
+    for (let i = 0; i < trackers.length; i++) {
+      const trackingId = trackers[i].get('trackingId');
+      if (trackingId === 'UA-50123418-16' || trackingId === 'UA-50123418-17') {
```

Comment:
I think this could be written more succinctly using some `Array` methods.

```
const vagovTrackingIds = [ ... ]; // Keeps any relevant tracking IDs in one place.

// Could be one- or two-lined if you wanted.
const tracker = trackers.find(tracker => {
  const trackingId = tracker.get('trackingId');
  return vagovTrackingIds.includes(trackingId);
});

const clientId = tracker && tracker.get('clientId');
```